### PR TITLE
Load logo from config

### LINKED ISSUE #319: Unable to load logo from config.yml
#### My reason:

I am specifying logo URL in the config, but not able to see the updated logo in web UI.

#### Steps to reproduce:
1. Uncomment **web -> logo** section in config and add a logo url.

####  Config file:	

I am using full.yaml file from config.

#### Additional information:
	
 - Container:
   - I use local environment

### PR DESCRIPTION
**Pull request by ayusharma**
**Type:** bug 

The following has been addressed in the PR:

*  There is a related issue #319 
*  Unit or Functional tests are included in the PR - All test passed.

**Description:**

Resolves #319

### CODE REVIEW

**Comment by juanpicado** [comment]
thanks 👍  @ayusharma I'll try to review it soon

File: src/api/web/index.js [hunk 1]
```diff
@@ -32,9 +32,8 @@ module.exports = function(config, auth, storage) {
     });
   });
 
-  router.get('/-/logo', function(req, res) {
-    res.sendFile(_.get(config, 'web.logo') || `${env.APP_ROOT}/static/logo-sm.png`
-    );
+  router.get('/-/verdaccio/logo', function(req, res) {
```
Comment (juanpicado) [inline src/api/web/index.js]: Why the `/-/verdaccio/` ?

File: src/webui/src/components/Header/index.js [hunk 2]
```diff
@@ -37,6 +38,16 @@ export default class Header extends React.Component {
     });
   }
 
+  componentWillMount() {
+    API.get('logo')
+    .then((response) => {
+      this.setState({logo: response.data});
+    })
+    .catch(() => {
```
Comment (juanpicado) [inline src/webui/src/components/Header/index.js]: It would be great to display the real error here.

File: src/api/web/index.js (hunk 1 above)
Comment (ayusharma) [reply src/api/web/index.js]: We have set 
https://github.com/verdaccio/verdaccio/blob/9cde5adfd4546c7c96cb83f108861b89e41e92ef/src/webui/template/index.html#L9

So every request from web UI is starting with it.

Comment (juanpicado) [reply src/api/web/index.js]: That's true, I didn't see that.

**Comment by juanpicado** [comment]
Could you rebase your branches onto master?

**Comment by ayusharma** [comment]
I have rebased it with the master.

**Pull request merged**